httpobs/scanner/grader/grade.py

```diff
@@ -22,7 +22,28 @@
     0: 'F'
 }
 
+# See https://wiki.mozilla.org/Security/Standard_Levels for a definition of the risk levels
+# We cannot make an accurate decision on HIGH and MAXIMUM risk likelihood indicators with the current checks,
+# thus the likelihood indicator is currently at best (or worse) MEDIUM
+LIKELIHOOD_INDICATOR_CHART = {
+    'A+': 'LOW',
+    'A': 'LOW',
+    'A-': 'LOW',
+    'B+': 'MEDIUM',
+    'B': 'MEDIUM',
+    'B-': 'MEDIUM',
+    'C+': 'MEDIUM',
+    'C': 'MEDIUM',
+    'C-': 'MEDIUM',
+    'D+': 'MEDIUM',
+    'D': 'MEDIUM',
+    'D-': 'MEDIUM',
+    'F': 'MEDIUM'
```

Comment:
To make this cleaner, I might recommend just having A, B, C, D, and F and then use grade[0].  Sorry for all this review, I really want to keep this code base tidy.  :)